File: server/entities/Eng.js

module.exports = class Eng{
    constructor(
        eng_id,
        group_id,
        pronunciation,
        content,
        type,
        path_image,
        listVN = [],
        listUser = []

    ){
        this.eng_id = eng_id;
        this.group_id = group_id;
        this.pronunciation = pronunciation;
        this.content = content;
        this.type = type;
        this.path_image = path_image;
        this.listVN = listVN;
        this.listUser = listUser;
    }
}

File: app/src/main/java/com/nhom2/bedatabase/domain/models/Eng.kt
package com.nhom2.bedatabase.domain.models

data class Eng(
    var eng_id: Int? = null,
    var group_id: Int,
    var pronunciation: String,
    var content: String,
    var type: String,
    var path_image: String? = null,
    var vns: List<Vn> = listOf()
)
